src/views/DashboardOrganization.js

```diff
@@ -1,30 +1,36 @@
-import { Container, IconButton } from '@material-ui/core';
+import {
+  Box,
+  ButtonGroup,
+  Container,
+  Grid,
+  IconButton,
+} from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import clsx from 'clsx';
 import React, { Fragment, useState } from 'react';
-import { useSelector } from 'react-redux';
-import { Link, generatePath } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
 
-import {
-  DASHBOARD_PATH,
-  MY_PROFILE_PATH,
-  ORGANIZATION_MEMBERS_PATH,
-  QR_GENERATOR_PATH,
```

Comment:
Let's not forget Lucia didnt want to remove this, she forgot about it in the new design.

Reply:
Yes but I don't think so we should include that info in code base. Lucia mentioned that once she has solution she will show it to us - would you like to keep track of it somehow?

Reply:
No but we should leave it up to her to remember I think